core/trigFilters.fcl

```diff
@@ -1772,6 +1772,16 @@ TrigCalFilters:{
     diagLevel: 0
     module_type: "CaloClusterCounterFilter"
   }
+  
+  caloHitRecN0SourceFilter: {
+        caloDigiCollection: "CaloDigiMaker"
+        MaxDigiADC: 1e6 // set higher for completeness
+        MinDigiADC: 0 //should we set a minimum
+        MaxNCry: 3// max number of crystals 
+        MinNCry: 1 // min number of crystals
+        diagLevel: 0
+        module_type: "CaloDigiSourceFilter"
```

Comment:
is this module already present to the HEAD of the Offline?

Reply:
no not in HEAD. I can put it in when we are OK with everything in this repo (I want to make sure I'm not doing anything totally wrong here before I put in that PR too)